### PR TITLE
New interactive content header

### PR DESCRIPTION
**Pull request by phamann**
As per discussions with the interactive team, this is a new clean and slimmed down header area for the default interactive template. Thanks to @OliverJAsh for starting the work.
### Desktop

google chrome
### Tablet

google chrome
### Mobile

google chrome

/cc @mattosborn @fcage

### CODE REVIEW

**Comment by OliverJAsh** [comment]
Sponsored content?

**Comment by OliverJAsh** [comment]
No byline?

**Comment by phamann** [comment]
@OliverJAsh Byline has moved to the bottom of the content, as requested by the interactive team.

File: common/app/views/fragments/headDefault.scala.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
             <h1 class="content__headline js-score" itemprop="headline">@Html(content.headline)</h1>
 
-            @if(!content.delegate.isVideo && !content.delegate.isGallery) {
+            @if(!content.delegate.isVideo && !content.delegate.isGallery && !content.isInteractive) {
```
Comment (gklopper) [inline common/app/views/fragments/headDefault.scala.html]: At this point you might want to abstract it...

**Comment by OliverJAsh** [comment]
:+1:

File: static/src/stylesheets/module/content/_interactive.scss [hunk 2]
```diff
@@ -62,31 +44,24 @@
...
     }
     .content__meta-container {
         margin-left: 0;
         width: 100%;
+        border-top: 0;
         border-bottom: 0;
 
+        .meta__number {
+            border-left: 1px solid colour(neutral-5);
+            padding-left: $gs-gutter / 2;
```
Comment (sammorrisdesign) [inline static/src/stylesheets/module/content/_interactive.scss]: Not vital, but the way this partial switches between `$gs-whatever/2` and `$gs-whatever / 2` is gross.

File: static/src/stylesheets/module/content/_interactive.scss [hunk 3]
```diff
@@ -163,4 +128,17 @@
     [data-alt="Live stream"] {
         @extend %u-responsive-ratio;
     }
+
+    .content__meta-footer {
+        .meta__email,
+        .meta__twitter {
+            display: inline !important;
+        }
```
Comment (sammorrisdesign) [inline static/src/stylesheets/module/content/_interactive.scss]: Same again, not vital but above in the file you've left an empty line between selectors, whereas here their isn't. eww

**Pull request merged**